lib/t/push_notifications/apns.ex

```diff
@@ -77,10 +77,13 @@ defmodule T.PushNotifications.APNS do
     base_alert_payload(type, alert)
   end
 
-  def build_alert_payload("match_about_to_expire" = type, _data) do
+  def build_alert_payload("match_about_to_expire" = type, data) do
+    %{"name" => name} = data
+
     alert = %{
-      "title" => dgettext("apns", "Your match is about to expire 😢"),
-      "body" => dgettext("apns", "Invite your match to a date if you want to keep it alive ✨")
+      "title" => dgettext("apns", "Your match with %{name} is about to expire 😢", name: name),
+      "body" =>
+        dgettext("apns", "Invite your match %{name} to a date if you want to keep it alive ✨")
```

Comment:
Putting name twice in the push doesn't sound right:

Your match with Dasha is about to expire 😢
Invite your match Dasha to a date if you want to keep it alive ✨